### DISCUSSION TITLE
Animation not working. Carousel, modal anything.

### DISCUSSION THREAD

**Discussion by RobertoLanzeni** [twbs/bootstrap#37581] [Q&A]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

Animation not working. Carousel, modal anything. 
Why?
I'm using bootstrap latest version  and live server extension on VSC. 

### Reduced test cases

```
<link
            href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
            rel="stylesheet"
            integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65"
            crossorigin="anonymous"
        />
<script
        src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"
        integrity="sha384-kenU1KFdBIe4zVF0s0G1M5b4hcpxyD9F7jL+jjXkk+Q2h455rYXK/7HAuoJl+0I4"
        crossorigin="anonymous"
    ></script>
<div id="main-caroisel" class="carousel slide" data-bs-ride="carousel">
            <div class="carousel-inner">
                <div class="carousel-item active c-item">
                    <img
                        src="assets/img/01.jpg"
                        class="d-block w-100 c-img"
                        alt="slide-1"
                    />
                </div>
                <div class="carousel-item c-item">
                    <img
                        src="assets/img/02.jpeg"
                        class="d-block w-100 c-img"
                        alt="slide-2"
                    />
                </div>
                <div class="carousel-item c-item">
                    <img
                        src="assets/img/01.jpg"
                        class="d-block w-100 c-img"
                        alt="slide-3"
                    />
                </div>
            </div>
        </div>
```

### What operating system(s) are you seeing the problem on?

Linux

### What browser(s) are you seeing the problem on?

Chrome, Firefox

### What version of Bootstrap are you using?

v5.2.3

**Comment by RobertoLanzeni** [comment]
No animation working. Any kind of default animation. No work

**Comment by mdo** [comment]
Assuming you mean transitions aren't working, you may have reduced motion enabled on your device. We can't help much here because you haven't done what the issue template asked—always provide a live example, not just code, of your issue.

**Reply by RobertoLanzeni** [reply]
I fix problem by downgrading my Linux distro from Linux mint 21 to 20.3, now animation works good! 
Thanks for yours support. 
Roberto
